module/VuFind/src/VuFind/View/Helper/Root/RecordDataFormatterFactory.php

```diff
@@ -253,9 +267,17 @@ public function getDefaultDescriptionSpecs()
         $spec->setLine('Awards', 'getAwards');
         $spec->setLine('Production Credits', 'getProductionCredits');
         $spec->setLine('Bibliography', 'getBibliographyNotes');
-        $spec->setLine('ISBN', 'getISBNs');
-        $spec->setLine('ISSN', 'getISSNs');
-        $spec->setLine('DOI', 'getCleanDOI');
+        $spec->setLine(
+            'ISBN', 'getISBNs', null,
+            ['itemPrefix' => '<span property="isbn">', 'itemSuffix' => '</span>']
+        );
+	$spec->setLine('ISSN', 'getISSNs', null,
+            ['itemPrefix' => '<span property="issn">', 'itemSuffix' => '</span>']
+	);
+	$spec->setLine(
+            'DOI', 'getCleanDOI', null,
+            ['prefix' => '<span property="identifier">', 'suffix' => '</span>']
```

Comment:
Is there a reason why you're using prefix/suffix here rather than itemPrefix/itemSuffix? Just because this is a single-valued field? Would it be better to use the item-level settings for consistency and in case of multiple DOIs in a custom instance?

Reply:
You're right! I will change it for "DOI" change it for "Edition" for consistency reasons.